package.json

```diff
@@ -75,7 +75,6 @@
     "index.d.ts",
     "index.d.cts",
     "dist",
-    "lib",
```

Comment:
I'd recommend leaving `lib` here to make things easier for VSCode debugging. See https://github.com/js-temporal/temporal-polyfill/pull/194#issuecomment-1336265534.

Reply:
Here's the VS Code bug that's triggered by removing `lib` from the package: https://github.com/microsoft/vscode/issues/168013